packaging/ondemand.spec

```diff
@@ -211,7 +211,7 @@ touch %{buildroot}%{_sysconfdir}/ood/config/ood_portal.sha256sum
 %__cat >> %{buildroot}%{_sysconfdir}/sudoers.d/ood << EOF
 Defaults:apache !requiretty, !authenticate
 Defaults:apache env_keep += "NGINX_STAGE_* OOD_*"
-apache ALL=(ALL) NOPASSWD: /opt/ood/nginx_stage/sbin/nginx_stage
+apache ALL=(ALL) NOPASSWD:SETENV: /opt/ood/nginx_stage/sbin/nginx_stage
```

Comment:
Requires an edit to the sudo rule to allow for environment variables to be passed.